imagebuilder/pkg/imagebuilder/config.go

```diff
@@ -22,7 +22,7 @@ type Config struct {
 }
 
 func (c *Config) InitDefaults() {
-	c.BootstrapVZRepo = "https://github.com/justinsb/bootstrap-vz.git"
+	c.BootstrapVZRepo = "https://github.com/andsens/bootstrap-vz.git"
```

Comment:
Please be aware of https://github.com/amzn/amzn-drivers/issues/40 - not solved at the moment of writing, but the patch for https://github.com/amzn/amzn-drivers provided in the first link worked for me.

Reply:
@valdemon that's just a case of waiting for that patch to get pushed before triggering a build right, no action from our/my side needed if I understand correct?

Reply:
And I've rebased and there's nothing on my fork that is missing from upstream now (all my PRs have merged)